code/modules/food&drinks/drinks/drinks/bottle.dm

```diff
@@ -300,6 +300,11 @@
 /obj/item/weapon/reagent_containers/food/drinks/bottle/molotov/attackby(obj/item/I, mob/user, params)
 	if(is_hot(I) && !active)
 		active = 1
+		var/turf/bombturf = get_turf(src)
+		var/area/bombarea = get_area(bombturf)
+		message_admins("[key_name(user)]<A HREF='?_src_=holder;adminmoreinfo=\ref[user]'>?</A> has primed a [name] for detonation at <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[bombturf.x];Y=[bombturf.y];Z=[bombturf.z]'>[bombarea.name] (JMP)</a>.")
+		log_game("[key_name(user)] has primed a [name] for detonation at [bombarea.name] ([bombturf.x],[bombturf.y],[bombturf.z]).")
+		
```

Comment:
I read people saying they're very OP in IRC.

On 27 July 2015 at 20:15, MrStonedOne <notifications@github.com> wrote:

> In code/modules/food&drinks/drinks/drinks/bottle.dm
> <https://github.com/tgstation/-tg-station/pull/10886#discussion_r35513484>
> :
>
> > @@ -300,6 +300,11 @@
> >  /obj/item/weapon/reagent_containers/food/drinks/bottle/molotov/attackby(obj/item/I, mob/user, params)
> >  	if(is_hot(I) && !active)
> >  		active = 1
> > +		var/turf/bombturf = get_turf(src)
> > +		var/area/bombarea = get_area(bombturf)
> > +		message_admins("[key_name(user)]<A HREF='?_src_=holder;adminmoreinfo=\ref[user]'>?</A> has primed a [name] for detonation at <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[bombturf.x];Y=[bombturf.y];Z=[bombturf.z]'>[bombarea.name] (JMP)</a>.")
> > +		log_game("[key_name(user)] has primed a [name] for detonation at [bombarea.name] ([bombturf.x],[bombturf.y],[bombturf.z]).")
> > +		
>
> you don't even need to do name, as byond converts reference types in [] to
> a string containing its name. so [get_area(bombturf)] works.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tgstation/-tg-station/pull/10886/files#r35513484>.
>